README.md

```diff
@@ -95,6 +95,23 @@ Apart from Events and Commands Refund Plugin is also based on mechanisms derived
 
 Configuration of all elements mentioned above can be found and customized in `config.yml` file.
 
+## Payment requirements
+
+By default to refund your order you need to have at least one available payment method configured as offline gateway.
+In case your custom refund logic demands different type of gateway (for example stripe), you need to modify parameters
+used in `src/Resources/config/services/provider.xml`
```

Comment:
By default, to refund your order, you need to have at least one available payment method configured with an `offline` gateway. In case your custom refund logic allows a different type of gateway (for example `stripe`), you should modify the specific parameter, as shown below: